### PR TITLE
braintree-web - Make ThreeDSecureVerifyOptions.addFrame optional

### PR DESCRIPTION
**Pull request by satansdeer**
The new Braintree 3DSecure implementation makes both `addFrame` and `removeFrame` optional. Here is the link to the [docs](https://developers.braintreepayments.com/guides/3d-secure/migration/javascript/v3#can-merchants-customize-the-presentation-of-the-bank-challenge?).

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.braintreepayments.com/guides/3d-secure/migration/javascript/v3#can-merchants-customize-the-presentation-of-the-bank-challenge?
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/braintree-web/test/web-new3ds.ts [hunk 1]
```diff
@@ -0,0 +1,794 @@
+import * as braintree from 'braintree-web';
```
Comment (jbuckner) [inline types/braintree-web/test/web-new3ds.ts:1]: What is the difference between this new test file and the current [`web.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/braintree-web/test/web.ts) test file? It looks like there's a lot of duplication between them. Could you just add your new test to that file?

Comment (satansdeer) [reply types/braintree-web/test/web-new3ds.ts:1]: yes sure, will remove the redundant parts

Comment (satansdeer) [reply types/braintree-web/test/web-new3ds.ts]: All right, done

**Comment by satansdeer** [comment]
Hey @jbuckner, thanks for the review, I've removed the duplications from the tests and all the checks are now passing. Are we ready to merge?

**Review by jbuckner — APPROVED** [review]

**Comment by satansdeer** [comment]
Ready to merge

**Pull request merged**